### PR TITLE
Add support for the VisioBraille Vario 4 Braille Device

### PR DESCRIPTION
**Pull request by Tobias-Team-Visiobraille**
### Link to issue number: 
None 

### Summary of the issue:
Set new PID for the Vario 4 series

### Description of how this pull request fixes the issue:
want to build 3 length of the Vario 4 20 40 80 because the device tell the screenreader the length while connection
we need only need one new PID

### Testing strategy:

Manual testing

Build (for testing PR): https://ci.appveyor.com/api/buildjobs/0djn47pgvabr30ut/artifacts/output/nvda_snapshot_pr12607-23270,2d86e96c.exe

### Known issues with pull request:

None

### Change log entries:
New features

```t2t
- Add support for the VisioBraille Vario 4 Braille Device. (#12607)
```

### Code Review Checklist:

- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual testing.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers

### CODE REVIEW

**Comment by Tobias-Team-Visiobraille** [comment]
@ the first commit the VID wasn´t correcht 0409 is right...the Alias VisioBraille should be used is that possible? @michaelDCurran

**Comment by XLTechie** [comment]
Do you think you might fill out the pull request template?

**Comment by seanbudd** [comment]
I've converted this to a draft, let us know when the template has been filled out and this is ready for review.

**Comment by feerrenrut** [comment]
On first inspection this looks like a duplicate of #12606, however the "VID" (vender ID) is different `VID_0483` vs `VID_0904` in #12606 vs #12607 respectively.

The only information given at the top of #12606 is:
> Want to insert our new Braille line "Vario 4" 6301 the lenght will be communicate after connection is esthablished. Hope this works with BT too.

Since these two PR's are so similar, I'm going to close #12606.
@Tobias-Team-Visiobraille we'll need more information and confirmation of testing results before we can include this change.

File: source/bdDetect.py [hunk 1]
```diff
@@ -413,6 +413,7 @@ def driverSupportsAutoDetection(driver):
 	"VID_0904&PID_600A", # Brailliant2 80
 	"VID_0904&PID_6201", # Vario 340
 	"VID_0483&PID_A1D3", # Orbit Reader 20
+	"VID_0904&PID_6301", # Vario 4
```
Comment (seanbudd) [inline source/bdDetect.py:416]: This will make our linter happy

```suggestion
	"VID_0904&PID_6301",  # Vario 4
```

**Review by seanbudd — CHANGES_REQUESTED** [review]
I've updated the title and the PR description.

You can test that this works using the build link you have added to the Pull Request description.

This pull request is currently targeting master, likely to be included in the 2021.3 release, if you wish this to be included in 2021.2 please test this and update the target branch for this request to `beta`.

>the Alias VisioBraille should be used is that possible? 

Is this something that is also blocking the request from being merged?

**Review by seanbudd — APPROVED** [review]

**Comment by Tobias-Team-Visiobraille** [comment]
Hello Sean, is there any possibility to include the Vario 4 device support into Update 2021.2 instead of 2021.3?
Thanks Tobias

**Comment by Dirk-Team-VisioBraille** [comment]
Hello Sean, 
this is Dirk from the VisioBraille Team. Since Tobias left the company its my job to organize the integration of the Braille displays into the screenreaders. I'm no software guy at all and I don't know how to manage this. Can you pleas help me?

What do I have to do for the integration of the Vario464 and the Vario480? 

Many thanks in advance for helping me.
Dirk

**Comment by Dirk-Team-VisioBraille** [comment]
@seanbudd 
Can you please help me?
Kind regards 
Dirk

**Comment by seanbudd** [comment]
@Dirk-Team-VisioBraille

Hi Dirk,
This is a fairly complicated request. I'd like to avoid discussing this on an unrelated pull request. Please send an email to info@nvaccess.org.

**Comment by Dirk-Team-VisioBraille** [comment]
Thanks for responding. I sent a mail for you to info@nvaccess.org. Will you respond it?

**Pull request merged**